golang/examples/mongodb.go

```diff
@@ -0,0 +1,74 @@
+package examples
+
+import (
+  "fmt"
+  "context"
+  "time"
+  mongoPsh "github.com/platformsh/config-reader-go/v2/mongo"
+  psh "github.com/platformsh/config-reader-go/v2"
+  "go.mongodb.org/mongo-driver/mongo"
+  "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
```

Comment:
Text alignment issue.  You have to run `go fmt` on a given directory; it won't run on a whole tree, unfortunately.

Reply:
boo